src/N98/Magento/Command/System/Cron/AbstractCronCommand.php

```diff
@@ -50,6 +57,7 @@ public function inject(
         $this->_cronConfig = $cronConfig;
         $this->_scopeConfig = $scopeConfig;
         $this->_cronScheduleCollection = $cronScheduleCollection;
+        $this->_timezone = $timezone;
```

Comment:
Please convert ->_... properties to ones w/o "_" at the beginning. I know it's now all your code, so please at least do it for new introduced properties even if code-style changes that way (it changes to the correct way).